modules/sequence-diagram-editor/js/main_elements/definitions/life-lines/resource.js

```diff
@@ -109,6 +109,13 @@ var MainElements = (function (mainElements) {
                         value: inputs.post.checked
                     }
                 ];
+            },
+            textModel: function(txtModel){
```

Comment:
Shall we enclose this textModel function inside utils? Then there will be no need to pass this as an additional parameter